log/src/io/pedestal/internal.clj

```diff
@@ -19,3 +20,33 @@
 (defn into-vec
   [v coll]
   (into (or v []) coll))
+
+(defn println-err [& s]
+  (binding [*out* *err*]
+    (apply println s)))
+
+(defn resolve-var-from
+  [property-name env-var]
+  (let [resolver (fn [from-source from-name s]
+                   (when s
+                     (try
+                       (let [slashx     (str/index-of s "/")
+                             ns-str     (when slashx
+                                          (subs s 0 slashx))
+                             symbol-str (if slashx
+                                          (subs s (inc slashx))
+                                          s)
+                             sym        (symbol ns-str symbol-str)
+                             v          (requiring-resolve sym)]
+                         (if v
+                           @v
+                           (println-err (format "WARNING: Symbol %s (from %s %s) does not exist"
+                                                s
+                                                from-source from-name))))
+                       (catch Exception e
+                         (println-err (format "ERROR: Could not resolve symbol %s (from %s %s): %s"
+                                              s
+                                              from-source from-name
+                                              (ex-message e)))))))]
+    (or (resolver "JVM property" property-name (System/getProperty property-name))
+        (resolver "environment variable" env-var (System/getenv property-name)))))
```

Comment:
`(resolver "environment variable" env-var (System/getenv property-name))` should be 
`(resolver "environment variable" env-var (System/getenv env-var))` right?

Reply:
Good catch!  Manual testing only goes so far, and environment variables are immutable globals.